bVNC/src/main/java/com/iiordanov/bVNC/RemoteCanvasActivity.java

```diff
@@ -279,19 +279,26 @@ public void run() {
                 } catch (NullPointerException e) {
                 }
             }};
+        Runnable clearToggles = new Runnable() {
```

Comment:
Please rename clearToggles runnable and function to something more descriptive to its function.